pkg/utils/globalTypes/Day/day.go

```diff
@@ -0,0 +1,21 @@
+package day_type
+
+type DayInformation struct {
+	IterationsPerDay       int
+	SimulationDays         int
+	DaysPerReshuffle       int
+	IterationsPerReshuffle int
+	TotalIterations        int
+	CurrTick               int
```

Comment:
```suggestion
	TicksPerDay       int
	SimulationDays         int
	DaysPerReshuffle       int
	TicksPerReshuffle int
	TotalTicks        int
	CurrTick               int
```

Reply:
Swapping between iterations and ticks gets confusing - better to just call them Ticks I think

Reply:
Obviously will need to change this wherever iteration has been used